MediaBrowser.Plugins.Trailers/Providers/ML/BaseProvider.cs

```diff
@@ -243,19 +244,19 @@ private string GetImdbId(IEnumerable<HtmlNode> nodes)
             return null;
         }
 
-        private List<ChannelMediaInfo> GetMediaInfo(IEnumerable<HtmlNode> nodes, string html)
+        private List<MediaSourceInfo> GetMediaInfo(IEnumerable<HtmlNode> nodes, string html)
         {
             var links = nodes.Select(i => i.GetAttributeValue("href", ""))
                 .Where(i => !string.IsNullOrWhiteSpace(i))
                 .ToList();
 
-            var list = new List<ChannelMediaInfo>();
+            var list = new List<MediaSourceInfo>();
 
             foreach (var link in links)
             {
                 if (ValidContainers.Any(i => link.EndsWith(i, StringComparison.OrdinalIgnoreCase)))
                 {
-                    //_logger.Debug("Found url: " + link);
+                    //_logger.LogDebug("Found url: {url}", link);
```

Comment:
You probably did a search and replace, but maybe remove this line

Reply:
I also do the comments just in case anyone wants to uncomment them. I'll be happy to remove the comment though

Reply:
removing